Android/app/src/main/java/objects/ChecklistItem.java

```diff
@@ -1,30 +1,41 @@
 package objects;
 
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.Ignore;
+import android.arch.persistence.room.PrimaryKey;
+
 /**
  * Model class for city object
  */
+@Entity(tableName = "events_new")
 public class ChecklistItem {
 
-    private final String mId;
+    //so that primary key is generated automatically
+    @PrimaryKey(autoGenerate = true)
+    @ColumnInfo(name="id")
+    private  int mId = 0;
+
+    @ColumnInfo(name="name")
     private final String mName;
+
+    @ColumnInfo(name="isDone")
     private final String mIsDone;
 
     // TODO :: Make isDone bool
```

Comment:
Integrate this thing also. Change `isDone` from `String` to `bool`

Reply:
@Swati4star umm There is no native boolean data type for SQLite. So if I change it to bool, it will create problems

Reply:
Leave it as is then.